partition_gpu/partition_gpu.go

```diff
@@ -98,18 +107,27 @@ func main() {
 		os.Exit(1)
 	}
 	if !migModeEnabled {
+		glog.Infof("Checking the GPU type now.")
+		gpuType, err := checkGpuType()
+		if err != nil {
+			glog.Errorf("Failed to check GPU Type: %v", err)
+			os.Exit(1)
+		}
+		glog.Infof("Got GPU type used: %s", gpuType)
 		glog.Infof("MIG mode is not enabled. Enabling now.")
```

Comment:
Move `glog.Infof("MIG mode is not enabled. Enabling now.")` before  `glog.Infof("Checking the GPU type now.")`

Reply:
Done